src/sentry/snuba/sessions.py

```diff
@@ -1035,110 +1035,3 @@ def __get_scope_value_for_release(
     elif scope == "crash_free_users":
         scope_value = rq_row["users_crashed"] / rq_row["users"]
     return scope_value
-
```

Comment:
Now that I see the diff, I am actually in favor of leaving the functions in this file untouched, and calling them from the service implementation.

Reply:
yup, also in favor of this. we can add typing at a later point